example/simple/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// The simple command demonstrates a simple functionality which
+// prompts the user for a GitHub username and lists all the public
+// organization memberships of the specified username.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+)
+
+// Fetch all the public organizations' membership of a user.
+//
+func FetchOrganizations(username string) ([]*github.Organization, error) {
+	ctx := context.Background()
+	client := github.NewClient(nil)
+
+	orgs, _, err := client.Organizations.List(ctx, username, nil)
+
+	if err != nil {
```

Comment:
I'd suggest dropping this `if` statement, just do:

```Go
orgs, _, err := client.Organizations.List(ctx, username, nil)
return orgs, err
```

It's simpler code and functionally the same. If `err` is non-nil, we can just pass `orgs` through unmodified. There's no need to explicitly return `nil` instead.

Reply:
I used to write such naive code 10 days ago!
I still do but I am definitely learning a lot.